internal/controller/ansibleRun/ansibleRun.go

```diff
@@ -262,15 +262,17 @@ func (c *connector) Connect(ctx context.Context, mg resource.Managed) (managed.E
 	// Requirements is a list of collections/roles to be installed, it is stored in requirements file
 	requirementRolesStr := string(requirementRoles)
 	if pc.Spec.Requirements != nil || requirementRolesStr != "" {
-		var requirementsType string
+		var installCollections bool
```

Comment:
`var installCollections, installRoles bool` for a better readability ;)

Reply:
cc @ron1

Reply:
Hi @fahedouch,  my apologies for the delay.